include/dem/pw_contact_force.h

```diff
@@ -114,6 +124,14 @@ class PWContactForce
     // Calculation of total force
     Tensor<1, dim> total_force = normal_force + tangential_force;
 
+    if (calculation_force_torque == true)
+      {
+        ForceOnWall[boundary_id] = ForceOnWall[boundary_id] - total_force;
+        TorqueOnWall[boundary_id] =
+          TorqueOnWall[boundary_id] -
+          cross_product_3d(point_on_boundary - center_mass, total_force);
```

Comment:
Will this always work in 3D? Aka is this always a 3D vector? Have you tested in 2D that this does not crash?

Reply:
The function description says that it will work in 2D. Didn't try, i will.